src/main/java/org/fenixedu/academic/dto/student/DomainObjectDeletionBean.java

```diff
@@ -0,0 +1,29 @@
+package org.fenixedu.academic.dto.student;
```

Comment:
@joaofreitas-qubit let's repackage this, the bean can be used for any DomainObject so let's have it in `org.fenixedu.academic.dto`